src/components/cards/LoginCard.vue

```diff
@@ -42,7 +54,50 @@
         </h1>
       </v-col>
     </v-row>
+    <!-- Maintenance notice -->
+    <div v-if="isMaintenancePeriod">
```

Comment:
Nice to have: Wollen wir da noch einen roten Kasten drum herum anzeigen? So wie beim SPSHAlert?

Reply:
Gute Idee! 

![image](https://github.com/user-attachments/assets/8f491eaa-8e1f-4486-bf7f-49b09a2800df)